packages/@okta/i18n/src/properties/login.properties

```diff
@@ -930,6 +930,13 @@ oie.symantecVip.challenge.title = Verify with {0}
 oie.symantecVip.challenge.description = Enter the generated security code from the {0} app.
 oie.symantecVip.verify.passcode.label = Enter security code
 
+## YubiKey Authenticator
+oie.yubikey.authenticator.description = Verify your identity using YubiKey
+oie.yubikey.enroll.title = Set up YubiKey
+oie.yubikey.challenge.title = Verify with YubiKey
+oie.yubikey.description = Insert the YubiKey into a USB port and tap it to generate a verification code.
+oie.yubikey.passcode.label = Insert then tap your YubiKey
```

Comment:
These keys are consistent with our [mock files](https://www.figma.com/file/I1ebb1ApKB1j7EkIaAeWFH/OIE-Authenticators?node-id=570%3A1155), and are currently displaying slightly different text than V1.

@nikhilvenkatraman-okta - Can you remind me where we stand with v1 customizations in regards to the "Set up" vs. "Setup" conversation? If we're OK with using the existing wording (and are OK with removing the period), we can eliminate two extra keys and keep v1 customizations.